src/common/components/autoSuggest/AutoSuggest.tsx

```diff
@@ -182,35 +191,26 @@ const AutoSuggest: React.FC<Props> = ({
     }
   };
 
-  const announceAriaLiveSelection = ({
-    event,
-    val,
-  }: {
-    event: string;
-    val: string;
-  }) => {
-    setAriaLiveSelection(valueEventAriaMessage({ event, value: val, t }));
-  };
-
-  const selectOption = (newValue: AutoSuggestOption) => {
-    announceAriaLiveSelection({
-      event: 'select-option',
-      val: newValue.label,
-    });
-  };
+  const selectOption = (option: AutoSuggestOption) => {
+    ensureMenuIsClosed();
+    onChange(
```

Comment:
Very hard to understand nested ternary here. Maybe if statement would be easier to comprehend